Rakefile

```diff
@@ -4,4 +4,4 @@
 require_relative "config/application"
 
 Rails.application.load_tasks
-task default: %i(jasmine:ci lint brakeman)
+task default: %i(i18n_cov:ci jasmine:ci lint brakeman)
```

Comment:
Are we confident that the default task gets run after the tests?
If not, I think there's a risk the JSON file might be missing? 🤔

Reply:
So this is because:

   - [RSpec shoves itself into the default task](https://github.com/rspec/rspec-rails/blob/eb3377bca425f0d74b9f510dbb53b2a161080016/lib/rspec/rails/tasks/rspec.rake#L6)
   - We then attempt to redefine the task
   - [Rake merges the prerequisites together](https://github.com/ruby/rake/blob/4fe73ff6b19ea1d8490b0442c75fc3a53815c4cf/lib/rake/task.rb#L116)
   - [Ruby set union appears to append new elements](https://ruby-doc.org/core-2.7.0/Array.html#method-i-7C)

I've raised this, as it doesn't particularly block this PR: https://github.com/alphagov/content-publisher/pull/1951

Reply:
Thanks for the explanation 👏